frontend/src/api/UserAPI.js

```diff
@@ -6,10 +6,17 @@ const Api = axios.create({
   baseURL: baseUri,
   headers: {
     'Content-Type': 'application/json',
-    Authorization: `Bearer ${window.sessionStorage.getItem('token')}`,
   },
 });
 
+// eslint-disable-next-line
+export const getCurrentUser = () =>
+  Api.get('/users/current', {
+    headers: {
+      Authorization: `Bearer ${sessionStorage.getItem('token')}`,
```

Comment:
Why it was moved here?

Reply:
It's folly to initialize Axios with a token that may or may not be there. The token should be added to each request individually so it's fetched when the request is called